frontend/src/components/nav/navbar.js

```diff
@@ -56,6 +57,15 @@ class NavBar extends React.Component {
         }
     }
 
+    debounceSearch(func, delay) {
+        let inDebounce;
+        return (e) => {
```

Comment:
For arrow functions with only one argument, you don't need parenthesis.

return e => {